onboard-blink/cpp/README.md

```diff
@@ -1,872 +1,31 @@
+# Onboard Blink 
 
 
+## Introduction
+This is a simple sample you could use for a quick test.
 
+## What it is
+A blink sample, it turns the on-board LED on for one second, then off for one second, repeatedly.
 
+## Hardware requirements
 
+A board with an accesible GPIO pin (connected to an LED).
 
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
-  <link rel="dns-prefetch" href="https://assets-cdn.github.com">
-  <link rel="dns-prefetch" href="https://avatars0.githubusercontent.com">
-  <link rel="dns-prefetch" href="https://avatars1.githubusercontent.com">
-  <link rel="dns-prefetch" href="https://avatars2.githubusercontent.com">
-  <link rel="dns-prefetch" href="https://avatars3.githubusercontent.com">
-  <link rel="dns-prefetch" href="https://github-cloud.s3.amazonaws.com">
-  <link rel="dns-prefetch" href="https://user-images.githubusercontent.com/">
+## Supported boards
 
+This sample has been tested on
+- MinnowBoard Turbot 2.0
+- Up2 board
 
+The sample might need minor modifications depending on the board and shield you are using.
 
-  <link crossorigin="anonymous" href="https://assets-cdn.github.com/assets/frameworks-d7137690e30123bade38abb082ac79f36cc7a105ff92e602405f53b725465cab.css" integrity="sha256-1xN2kOMBI7reOKuwgqx582zHoQX/kuYCQF9TtyVGXKs=" media="all" rel="stylesheet" />
-  <link crossorigin="anonymous" href="https://assets-cdn.github.com/assets/github-15d10d5c5d04521a59aed66ac12ddf49a051df082e9488bf8241b716e792d414.css" integrity="sha256-FdENXF0EUhpZrtZqwS3fSaBR3wgulIi/gkG3FueS1BQ=" media="all" rel="stylesheet" />
-  
-  
-  
-  
+## Software requirements
+This version of the sample has been tested on Ubuntu Linux. It requires the [mraa library](https://github.com/intel-iot-devkit/mraa). 
 
-  <meta name="viewport" content="width=device-width">
-  
-  <title>iot-devkit-samples/README.md at v2 · intel-iot-devkit/iot-devkit-samples</title>
-  <link rel="search" type="application/opensearchdescription+xml" href="/opensearch.xml" title="GitHub">
-  <link rel="fluid-icon" href="https://github.com/fluidicon.png" title="GitHub">
-  <meta property="fb:app_id" content="1401488693436528">
 
-    
-    <meta content="https://avatars0.githubusercontent.com/u/7722711?s=400&amp;v=4" property="og:image" /><meta content="GitHub" property="og:site_name" /><meta content="object" property="og:type" /><meta content="intel-iot-devkit/iot-devkit-samples" property="og:title" /><meta content="https://github.com/intel-iot-devkit/iot-devkit-samples" property="og:url" /><meta content="iot-devkit-samples - Intel System Studio IoT Edition project code samples using the Intel® IoT Developer Kit libraries." property="og:description" />
+## Setup
+Create a new project on Intel(R) System Studio using this sample. Make sure the on board LED or the external LED is connected to the board.
```

Comment:
you are using `on board` here and `on-board` in other places. Use just one of those